modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,19 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you shoot with a {pulsar-x}:
+
+. Roll the dice listed in brackets, for example (D3+1) or (D6).
+. Resolve that many {macro-weapon} shots against the target detachment.
+
+IMPORTANT: Remember that a {pulsar-x} is a single super-heavy weapon.
+It does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, or contribution to firefights.
```

Comment:
Just a thought, and I'm not in any way wedded to this, but might a more concise version be slightly better?

```suggestion
It does *not* count as multiple super-heavy weapons for any game purpose.
```

or perhaps:

```suggestion
It does *not* count as multiple super-heavy weapons for any game purpose, such as placement of Blast markers.
```

If you think that the explicit list you already drafted is better for any reason — and you're confident that there aren't any other list items that we're forgetting — then we can leave this sentence as it is.

Reply:
I don't remember who drafted that bit originally, but I'd prefer to keep it as an explicit list. Mainly because it gets forgotten in firefights.

If anything, I'm inclined to make it more comprehensive by combining the old and new versions:

```suggestion
It does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

Reply:
> I don't remember who drafted that bit originally

Doesn't matter - we'll often think of new tweaks and ways to shape things when we're editing for one reason or another.

> but I'd prefer to keep it as an explicit list. Mainly because it gets forgotten in firefights.
>
>If anything, I'm inclined to make it more comprehensive by combining the old and new versions:

Fair enough. The 'or any other game purpose' bit covers my first concern — once I got on that track, I thought there was also an opportunity to make it more concise. But I'm happy to go with your suggestion here.